recognition/include/pcl/recognition/ransac_based/bvh.h

```diff
@@ -127,7 +127,8 @@ namespace pcl
             Node (std::vector<BoundedObject*>& sorted_objects, int first_id, int last_id)
             {
               // Initialize the bounds of the node
-              memcpy (bounds_, sorted_objects[first_id]->getBounds (), 6*sizeof (float));
+              auto sorted_bounds{sorted_objects[first_id]->getBounds()};
```

Comment:
Why did you create a temporary? It's not wrong and the compiler will optimize it away easily, just asking

Reply:
The original `std::copy` expression looked messy